lab-tara/app/src/main/java/com/example/chatroomstatus/models/Status.java

```diff
@@ -0,0 +1,17 @@
+package com.example.chatroomstatus.models;
+
+public class Status {
+    public enum STATUS {
+        ONLINE,
```

Comment:
Looks like you started down the path of using an enum for statuses. Definitely a good approach.